Mathlib/RingTheory/Polynomial/Chebyshev.lean

```diff
@@ -81,6 +82,24 @@ protected theorem induct (motive : ℤ → Prop)
   T.induct motive zero one add_two fun n hn hnm => by
     simpa only [Int.negSucc_eq, neg_add] using neg_add_one n hn hnm
 
+/-- Another induction principle, which is easier to use -/
```

Comment:
If it's always easier to use, we should probably replace the original. Otherwise, perhaps qualify as "sometimes easier to use".

Also, please mention that this is an induction principle for Chebyshev polynomials, similar to the docstring for `induct`